src/layouts/Basic.tsx

```diff
@@ -11,21 +11,23 @@ import { Navigation } from './Navigation'
 
 const Container = styled.div(
   ({ theme }) => css`
-    padding: ${theme.space['5']} ${theme.space['4']};
+    padding: ${theme.space['4']};
     display: flex;
-    flex-gap: ${theme.space['8']};
-    gap: ${theme.space['8']};
+    flex-gap: ${theme.space['4']};
+    gap: ${theme.space['4']};
     flex-direction: column;
     align-items: stretch;
     @supports (-webkit-touch-callout: none) {
       width: calc(100% - ${theme.space['8']});
       box-sizing: content-box;
-      ${mq.md.min(css`
+      ${mq.sm.min(css`
```

Comment:
this was a weird ipad hack for something i cant remember, can you check the layouts on ipad are fine